docs/source/GetStartedGuide.rst

```diff
@@ -65,6 +65,28 @@ Enterprise Linux.  Please refer to the `Avocado Long Term
 Stability`_ thread and to your package management docs on how to
 switch to the ``avocado-lts`` repo.
 
+OpenSUSE
+~~~~~~~~
+
+Avocado test framework has a [package in
+OpenSUSE](https://build.opensuse.org/package/show/Virtualization:Tests/avocado).
+You can install packages by running the following commands::
+
+    sudo zypper install avocado
+
+Ubuntu
+~~~~~~~~
+
+First, add PPA ppa:lmr/avocado with the package to your system's Software
+Sources by running the following command::
+
+    sudo add-apt-repository ppa:lmr/avocado
```

Comment:
@ligurio @lmr I need a hand here.  According to:

https://launchpad.net/~lmr/+archive/ubuntu/avocado

The packages are available for two series: willy and vivid.  Then I referred to those releases at:

https://launchpad.net/ubuntu/+series

And then tried to download them at:

http://releases.ubuntu.com/

And I could only find vivid:

http://releases.ubuntu.com/15.04/

Still, the only images are available are "snappy" ones, which don't use `deb` packages.  I'm really confused how/if the current packages are installable by regular users of official releases.